### PR TITLE
Updated documentation as per query conditions and spelling fixes

### PR DESCRIPTION
**Pull request by akumar-99**
### Checklist
- [ ] Issue(s) linked

### CODE REVIEW

**Comment by rajlearner17** [comment]
@akumar-99 Appreciate your feedback and the PR 👍 

Some of the content is from CIS doc directly as we don't change them, or we might have missed appropriate updates. We will carefully review it and bring it on. Again thanks for using Steampipe.

File: cis_v130/docs/cis_v130_5_1.md [hunk 1]
```diff
@@ -16,6 +16,6 @@ Public access to remote server administration ports, such as 22 and 3389, increa
 5. Choose the **Inbound Rules** tab and click **Edit Inbound rules**
 6. Check remote server access port entries. **Note** A Port value of ALL or a port range such as 0-1024 are inclusive of port 22, 3389, and other remote server administration ports.
 7. Either
-    - A) Update the `source` field to a range other than 0.0.0.0/0 or
+    - A) Update the `source` field to a range other than 0.0.0.0/0 or ::/0
```
Comment (cbruno10) [inline cis_v130/docs/cis_v130_5_1.md:19]:
```suggestion
    - A) Update the `source` field to a range other than 0.0.0.0/0 or
```

@akumar-99 I agree that this change is more helpful, but in this case, I don't believe we'll deviate from the official CIS documentation. Similar to my rationale in https://github.com/turbot/steampipe-mod-aws-compliance/issues/509#issuecomment-1263828637, it's often easier for us to stay with CIS and reduces possible confusion between our documentation and their's.

In terms of seeing this change in future versions, one suggestion I have is to submit a change or open a discussion on the CIS WorkBench website for the upcoming AWS v1.6.0 benchmark and propose this change.

File: cis_v130/docs/cis_v130_5_2.md [hunk 2]
```diff
@@ -13,8 +13,8 @@ Public access to remote server administration ports, such as 22 and 3389, increa
 3. For each security group to remediate, perform the following:
 4. Select the respective security group
 5. Choose the **Inbound Rules** tab and click **Edit Inbound rules**
-6. Identify the rules to be edited or removed bas on the remote server access port entries. **Note** A Port value of ALL or a port range such as 0-1024 are inclusive of port 22, 3389, and other remote server administration ports.
+6. Identify the rules to be edited or removed based on the remote server access port entries. **Note** A Port value of ALL or a port range such as 0-1024 are inclusive of port 22, 3389, and other remote server administration ports.
 7. Either
-    - A) Update the `source` field to a range other than 0.0.0.0/0 or
+    - A) Update the `source` field to a range other than 0.0.0.0/0 or ::/0
```
Comment (cbruno10) [inline cis_v130/docs/cis_v130_5_2.md:18]:
```suggestion
    - A) Update the `source` field to a range other than 0.0.0.0/0 or
```

File: cis_v130/section_5.sp [hunk 3]
```diff
@@ -21,7 +21,7 @@ benchmark "cis_v130_5" {
 }
 
 control "cis_v130_5_1" {
-  title         = "5.1 Ensure no Network ACLs allow ingress from 0.0.0.0/0 to remote server administration ports"
+  title         = "5.1 Ensure no Network ACLs allow ingress from 0.0.0.0/0 or ::/0 to remote server administration ports"
```
Comment (cbruno10) [inline cis_v130/section_5.sp:24]:
```suggestion
  title         = "5.1 Ensure no Network ACLs allow ingress from 0.0.0.0/0 to remote server administration ports"
```

File: cis_v130/section_5.sp [hunk 4]
```diff
@@ -35,7 +35,7 @@ control "cis_v130_5_1" {
 }
 
 control "cis_v130_5_2" {
-  title         = "5.2 Ensure no security groups allow ingress from 0.0.0.0/0 to remote server administration ports"
+  title         = "5.2 Ensure no security groups allow ingress from 0.0.0.0/0 or ::/0 to remote server administration ports"
```
Comment (cbruno10) [inline cis_v130/section_5.sp:38]:
```suggestion
  title         = "5.2 Ensure no security groups allow ingress from 0.0.0.0/0 to remote server administration ports"
```

File: cis_v150/docs/cis_v150_5_3.md [hunk 5]
```diff
@@ -6,14 +6,15 @@ Public access to remote server administration ports, such as 22 and 3389, increa
 
 ## Remediation
 
-Perform the following to implement the prescribed state:
+### From Console
```
Comment (cbruno10) [inline cis_v150/docs/cis_v150_5_3.md:9]: Can you please revert these changes as well? It is inconsistent with some of the other Remediation sections in these benchmarks, but since CIS has their documentation in this format, we choose to follow it as closely as we can.

**Review by cbruno10 — CHANGES_REQUESTED** [review]
@akumar-99 Thanks again for raising this PR!

I've commented on a few files with proposals to revert some of the changes you've made. Instead of retroactively updating all CIS docs, we choose instead to look forward to future CIS versions, like v1.6.0.

I didn't comment on every instance where I would suggest reverting, but can you please have a look at my comments and apply those across the PR? The general typo fixes we do definitely want to merge in though.

Thanks!

File: cis_v130/docs/cis_v130_5_1.md (hunk 1 above)
Comment (akumar-99) [reply cis_v130/docs/cis_v130_5_1.md:19]: @cbruno10 I have a favour to ask from you. Can you please help me with the procedure to propose a change on CIS WorkBench?

Comment (cbruno10) [reply cis_v130/docs/cis_v130_5_1.md]: @akumar-99 I'll see if we can propose additional changes in the next few weeks. Proposing changes on the site isn't difficult, but time consuming, so it's often a lower priority for us (among all other plugin + mod work).

Comment (akumar-99) [reply cis_v130/docs/cis_v130_5_1.md]: Got it. Thank you :)

File: cis_v140/docs/cis_v140_5_2.md [hunk 6]
```diff
@@ -13,8 +13,8 @@ Public access to remote server administration ports, such as 22 and 3389, increa
 3. For each security group to remediate, perform the following:
 4. Select the respective security group
 5. Choose the **Inbound Rules** tab and click **Edit Inbound rules**
-6. Identify the rules to be edited or removed bas on the remote server access port entries. **Note** A Port value of ALL or a port range such as 0-1024 are inclusive of port 22, 3389, and other remote server administration ports.
+6. Identify the rules to be edited or removed based on the remote server access port entries. **Note** A Port value of ALL or a port range such as 0-1024 are inclusive of port 22, 3389, and other remote server administration ports.
 7. Either
-    - A) Update the `source` field to a range other than 0.0.0.0/0 or
+    - A) Update the `source` field to a range other than 0.0.0.0/0 or ::/0
```
Comment (cbruno10) [inline cis_v140/docs/cis_v140_5_2.md:18]: Hey @akumar-99 , I'm not sure if you were planning on making additional changes, but can you please also revert any changes where you added `::/0`? Afterward, we can accept the PR, thanks!

**Review by cbruno10 — CHANGES_REQUESTED** [review]
@akumar-99 Please see additional comment, thanks!

File: cis_v140/docs/cis_v140_5_2.md (hunk 6 above)
Comment (akumar-99) [reply cis_v140/docs/cis_v140_5_2.md]: Reverted all the ::/0 changes

**Review by cbruno10 — APPROVED** [review]

**Comment by cbruno10** [comment]
Thanks @akumar-99 for catching and fixing these typos!
